site/src/components/steps.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Steps
+description: Steps the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com/design/7raji0IWkuBRxs5kLiADgI/Copilot-Web-UI-kit?node-id=70108-9255&node-type=frame&t=G0FoPLEXPee1yysa-0
+classType: Component
+classPrefixes:
+  - steps
```

Comment:
This also needs the `step` prefix, to catch the `.step` and `.step-selected` classes.

```suggestion
classPrefixes:
  - step
  - steps
```